src/components/calcite-action/calcite-action.scss

```diff
@@ -18,7 +18,7 @@
     justify-start
     m-0
     relative
-    text--2h
+    text-n2h
```

Comment:
classes that start with a dash were not working. They would either not be applied or cause a build error.

I changed all of them to be `n2h` instead of `-2h`.

https://github.com/tailwindlabs/tailwindcss/issues/6599